org.gnome.Rhythmbox3.json

```diff
@@ -43,13 +43,13 @@
         {
             "name": "grilo",
             "buildsystem": "meson",
-            "config-opts": [ "-Denable-gtk-doc=false", "-Denable-grl-pls=false" ],
+            "config-opts": [ "-Denable-gtk-doc=false", "-Denable-grl-pls=false", "-Dsoup3=false" ],
```

Comment:
I think you mean true here.

Reply:
But rhythmbox still depends on libsoup-2.4